apollo-model/src/main/java/io/logz/apollo/models/DeploymentMarker.java

```diff
@@ -0,0 +1,53 @@
+package io.logz.apollo.models;
+
+import java.util.Map;
+
+public class DeploymentMarker {
+    private String description;
+    private String title;
+    private Map<String, String> metadata;
+    private String tag = "DEPLOYMENT";
+    private String timestamp = null;
```

Comment:
timestamp is never used
either set the timestamp before sending marker or delete it entirely

Reply:
removed